packages/core/content-type-builder/server/src/services/architect.ts

```diff
@@ -151,12 +152,46 @@ const systemPrompt = `
           </examples>
     `;
 
+const applyDefaultValues = (schema: any) => {
```

Comment:
I am trying to fine tune the prompt but this seems like a good failsafe

Reply:
I hate having to do this :grimacing: 
But if we don't have a choice, why not

Reply:
Can't we cast it to unknown as that's what it is ?

Reply:
we do what we have for a successful demo

Reply:
🫡 amen